.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Ask a question or get support
+    url: https://github.com/fast-excel/fastexcel/discussions
+    about: Ask a question or request support for using Fastexcel
+  - name: 用户问题或提出建议
+    url: https://github.com/fast-excel/fastexcel/discussions
+    about: 反馈使用 Fastexcel 时遇到的问题，或对 Fastexcel 的建议
```

Comment:
User questions are different from bug reports and/or issues. Commonly user questions are related to usage, rather than anything that should be changed from the source code.